### PR TITLE
CB-3210. Refactor telemetry models (use objects instead of booleans)

### PR DESCRIPTION
**Pull request by oleewere**
- use objects for boolean flags (reportDeploymentLogs and meteringEnabled)
- removed databusEndpoint from WA request (only from request/response)
- remove logging/attributes from telemetry and replace it with fluentAttributes

### CODE REVIEW

File: common-model/src/main/java/com/sequenceiq/common/api/type/FeatureSetting.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+@ApiModel
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class FeatureSetting implements Serializable {
+
+    @NotNull
+    @ApiModelProperty(value = "enabled", required = true)
+    private Boolean enabled;
+
+    public Boolean getEnabled() {
```
[the remark this answers is not in the document]
Comment (oleewere) [reply common-model/src/main/java/com/sequenceiq/common/api/type/FeatureSetting.java]: was generated, i guess will work with that as well

File: common/src/main/java/com/sequenceiq/cloudbreak/telemetry/fluent/FluentConfigService.java [hunk 2]
```diff
@@ -76,13 +78,13 @@ public FluentConfigView createFluentConfigs(String clusterType, String platform,
...
         boolean validDatabusLogging = false;
-        if (meteringEnabled || telemetry.isReportDeploymentLogs()) {
+        if (meteringEnabled || isReportDeploymentLogsEnabled(telemetry)) {
             if (databusEnabled && meteringEnabled) {
                 builder.withMeteringEnabled(true);
                 LOGGER.debug("Fluent will be configured to send metering events.");
                 validDatabusLogging = true;
             }
-            if (databusEnabled && telemetry.isReportDeploymentLogs()) {
+            if (databusEnabled && isReportDeploymentLogsEnabled(telemetry)) {
```
[the remark this answers is not in the document]
Comment (oleewere) [reply common/src/main/java/com/sequenceiq/cloudbreak/telemetry/fluent/FluentConfigService.java]: if you assume that reportDeploymentLogsEnabled as a setting option, that should be "is" not "are"

[the remark this answers is not in the document]
Comment (attilapalfi92) [reply common/src/main/java/com/sequenceiq/cloudbreak/telemetry/fluent/FluentConfigService.java]: right

**Comment by oleewere** [comment]
retest this please

**Comment by akanto** [comment]
I am afraid the SQL has some compatibility issues, and during the rollout, it will break all exsisting pods.

**Pull request merged**